examples/classify_image.py

```diff
@@ -76,7 +76,7 @@ def main():
     raise ValueError('Only support uint8 input type.')
 
   size = common.input_size(interpreter)
-  image = Image.open(args.input).convert('RGB').resize(size, Image.ANTIALIAS)
+  image = Image.open(args.input).convert('RGB').resize(size, Image.LANCZOS)
```

Comment:
There're too many things going on in this one PR, changes like this should be in a separate cl.